src/entryPoints/HyperSwitchApp.res

```diff
@@ -134,13 +133,6 @@ let make = () => {
                         }}
                       />
                     </div>
-                    {switch url.path->urlPath {
-                    | list{"home"} =>
-                      <RenderIf condition=isLiveUsersCounterEnabled>
-                        <ActivePaymentsCounter />
```

Comment:
why have you removed the component rendering `<ActivePaymentsCounter /> `
you can just remove the feature flag and renderif condition